<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Point24_guiClass</class>
 <widget class="QWidget" name="Point24_guiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>916</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Point24_gui</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>30</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;24点计算器&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>80</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>请输入4个正整数（范围 1-13）</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="num1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>113</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="num2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>150</y>
     <width>113</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="num3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>150</y>
     <width>113</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>150</y>
     <width>113</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="calculate">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>150</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>计算</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>150</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>清除</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>250</y>
     <width>761</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>结果：</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Point24_gui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>calculate</sender>
   <signal>clicked()</signal>
   <receiver>Point24_guiClass</receiver>
   <slot>caculate_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>708</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>715</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clear</sender>
   <signal>clicked()</signal>
   <receiver>Point24_guiClass</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>858</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>902</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>caculate_button()</slot>
  <slot>clear()</slot>
 </slots>
</ui>
